src/main/java/org/terasology/dynamicCities/region/RegionEntityManager.java

```diff
@@ -75,18 +72,18 @@ public void postBegin() {
         }
         regionEntitiesComponent = new RegionEntitiesComponent(gridSize);
         regionStoreEntity = entityManager.create(regionEntitiesComponent, new RegionMainStoreComponent());
-        regionStoreEntity.setAlwaysRelevant(true);
+        regionStoreEntity.setScope(EntityScope.GLOBAL);
```

Comment:
I think @DarkWeird mentioned something about removing sectors (and therefore scopes?). Just to make people aware ...

Reply:
This change was because `setAlwaysRelevant` is deprecated, and `setScope` is what it pointed me to.

Reply:
Global scope exists yet.